ui/resizable.js

```diff
@@ -173,13 +174,14 @@ $.widget("ui.resizable", $.ui.mouse, {
 
 				handle = $.trim(n[i]);
 				hname = "ui-resizable-" + handle;
-				axis = $("<div class='ui-resizable-handle " + hname + "'></div>");
+				axis = $("<div>");
+				this._addClass( axis, "ui-resizable-handle " + hname );
 
 				axis.css({ zIndex: o.zIndex });
 
 				// TODO : What's going on here?
 				if ("se" === handle) {
-					axis.addClass("ui-icon ui-icon-gripsmall-diagonal-se");
+					this._addClass( axis, null, "ui-icon ui-icon-gripsmall-diagonal-se" );
```

Comment:
We should at least revisit this, not just because of the TODO. It seems like customizing icons would be a good usecase for the `classes` option, even though it overlaps with other `icons` options we have in other widgets.